### PR TITLE
Make operators upper and swap LIKE for CONTAINS

### PR DESCRIPTION
**Pull request by zeantsoi**
:eyeglasses: @dwick @bsimpson63 

According to Adzerk, the instances where they told us to use `LIKE` should actually be `CONTAINS`. I've tested it in the Adzerk UI and `CONTAINS` seems to work; I've made the change.

Also, it looks like the [Adzerk parser](https://github.com/adzerk/zerkel/blob/master/src/zerkel-parser.js#L638) accepts case-insensitive logical operators. I've switched all our operators to uppercase to better conform with general SQL conventions.

### CODE REVIEW

File: reddit_adzerk/adzerk_utils.py [hunk 1]
```diff
@@ -75,12 +75,12 @@ def get_mobile_targeting_query(os_str='',
     queries.append(os_query)
 
     if devices and versions:
-        device_queries = ['$device.%s like "%s"' % (lookup_str, device)
+        device_queries = ['$device.%s CONTAINS "%s"' % (lookup_str, device)
```
Comment (bsimpson63) [inline reddit_adzerk/adzerk_utils.py]: can this be a separate commit?

**Comment by zeantsoi** [comment]
:haircut:

**Comment by bsimpson63** [comment]
:fish:
